tests/testAstrometryMixins.py

```diff
@@ -224,7 +225,7 @@ def testUtilityMethods(self):
                  ('xPix', float), ('yPix', float),
                  ('xFocalPlane', float), ('yFocalPlane', float)]
 
-        baselineData = np.loadtxt(catName, dtype=dtype, delimiter=';')
+        baselineData = np.genfromtxt(catName, dtype=dtype, delimiter=';')
```

Comment:
Is there a reason you switched to genfromtxt here but not in VariabilityMixin.py?